conseil-lorre/src/main/resources/application.conf

```diff
@@ -99,4 +99,11 @@ lorre {
     interval: 360 minutes # interval between fetches
   }
 
+  fork-handling {
+    backtrack-levels: 100 //how many levels back we should check for the forks
+    backtrack-levels: ${?CONSEIL_LORRE_FORK_HANDLING_BAKCTRACK_LEVELS}
+    backtrack-interval: 120 // every how many iterations we should check for forks
```

Comment:
Are you sure 100 levels will be enough every 120 iterations?

Reply:
I've used default values from the #1045 but to be honest to get right values here is trickier than it seems.
For example:
Let's assume that blocks on depth 10 and greater from head are safe from forks.
We started Lorre which is fully indexed and there are no new blocks in the node. 
Backtracking forks check looks at the last 10 indexed blocks and everything is ok.
Then it waits number of intervals for a next check. Now we need to estimate amount of blocks that will be baked during this time.
And that depends on few things: `sleep_interval` and `backtrack-interval`.  
I believe with those assumptions we should make `backtrack-levels` to be ((Amount_of_blocks_baked_during(`sleep_interval`) + Amount_of_blocks_baked_during_processing) * `backtrack-interval`) + 10 + 1. 
And that should be an estimate. 
We can tune it afterwards, but I definitely think we should follow this rule: `backtrack-levels` > `backtrack-interval`